components/TxForm.tsx

```diff
@@ -1,3 +1,4 @@
+import useNotifications from "../hooks/useNotifications";
```

Comment:
let's stay consistent and use the `@` reference
```suggestion
import useNotifications from "@/hooks/useNotifications";
```

Reply:
Ah will fix and use in the future